docs/metal_interconnections_get.md

```diff
@@ -0,0 +1,44 @@
+## metal interconnections get
+
+Retrieves a list of interconnections.
+
+### Synopsis
+
+Retrieves a list of interconnections available to the default Organizations and by project ID
+
+```
+metal interconnections get [flags]
+```
+
+### Examples
+
+```
+  # Lists interconnections available to the default OrganizationID OR by project ID:	
+  metal interconnections get
```

Comment:
This should document the flags for specifying an organization ID or a project ID